packages/calcite-components/src/components/block/block.scss

```
@@ -11,6 +11,9 @@
  * @prop --calcite-block-heading-text-color-press: When the component is `expanded`, specifies the `heading` text color.
  * @prop --calcite-block-padding: [Deprecated] Specifies the padding of the component's `default` slot.
  * @prop --calcite-block-text-color: Specifies the component's text color.
+ * @prop --calcite-block-description-text-color: Specifies the component's description text color.
```

For doc consistency with prop reference 📚 :

```suggestion
 * @prop --calcite-block-description-text-color: Specifies the component's `description` text color.
```